python/ray/tune/trial.py

```diff
@@ -333,8 +333,8 @@ def restore_from_obj(self, obj):
     def update_last_result(self, result, terminate=False):
         if terminate:
             result = result._replace(done=True)
-        if terminate or (
-                self.verbose and
+        if self.verbose and (
```

Comment:
shouldn't we always print the result if it terminated?

Reply:
in testing out HyperOpt, I wanted a way to only get the debugstr (that listed all the trials at once) as it's clean and it's the only thing I needed. 

Even when printing out only results of terminated trials, stdout would be flooded.

Reply:
Ic, let's try it then.